atlassian-prod/etc/description.mustache

```diff
@@ -1,22 +1,20 @@
-The snowdrop team is really pleased to contact you as we will release in 12 weeks a new Snowdrop BOM based on the following Spring Boot version: *{{release.version}}*
+The Snowdrop team is really pleased to inform you that we're planning to release a new Snowdrop BOM on {{release.date}} based on Spring Boot *{{release.version}}*.
 
-{{#isComponent}}
-As product owner of the following {{type}}: *{{component.name}}*, we would like to know if you plan to release a new {{type}} for the version: *{{component.version}}*
-that we will start to test the: *{{release.dueDateFormatted}}*.
-{{/isComponent}}
-{{#isStarter}}
-As product owner of the following {{type}}: *{{component.name}}*, we would like to know if you plan to release a new {{type}}
-that we will start to test the: *{{release.dueDateFormatted}}*.
-{{/isStarter}}
+We, however, cannot release a new version without your support as Red Hat's Spring Boot stack relies on supported versions for the following artifact(s) 
+that are currently owned by your project:
+ * {{component.name}} 
+ * artifact 2 TODO
```

Comment:
What is the purpose of this text `artifact 2 TODO` ? @metacosm

Reply:
I need to look at the actual code feeding the template but ideally, we would pass the list of all relevant artifacts instead of just one component as shown in the JIRA ticket.

Reply:
> we would pass the list of all relevant artifacts instead of just one component as shown in the JIRA ticket.

As the description is tailored for each component or starter, why do you want to pass the full list to each JIRA project (EAP, JWS, ...) ? @metacosm

Reply:
To be clear about what we need instead of just saying Hibernate needs an update when we don't really care about all of it.